### PR TITLE
Group bookings support

### PR DESCRIPTION
**Pull request by laander**
Adds general support for fetching group booking slots as availability and creating group customer bookings. Can be used in conjunction with the latest update https://admin.timekit.io where widgets can be setup in "Many to 1" mode.

**Features**:
- If `bookingGraph` is set to `group_customer`, we assume group bookings mode and fetch availability + create new bookings accordingly
- Fetch only group booking slots created for the specific widget
- If timeslot durations are under 90 minutes, then make the timeslots appear larger (double height) in order not to truncate text in the event

**Improvements**:
- Made `goToFirstEvent` a reusable method
- Save reference to widget ID when creating bookings
- Refetch availability when closing booking page after a successful booking
- Calendar is only required when not in group bookings mode
- Made tests for group booking cases

**Bugfixes**:
- Fix height issue when timezoneHelper was disabled

### CODE REVIEW

File: examples/group-bookings.htm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        email:    'timebirdcphtest@gmail.com',
+        apiToken: 'password',
+        calendar: '31078529-08db-4778-a21e-2bc8ddff806e',
+        avatar:   '../misc/avatar-doc.jpg',
+        bookingGraph: 'group_customer',
+        timekitConfig: {
+          app: 'admin',
+          apiBaseUrl: 'http://api-localhost.timekit.io/'
+        }
+        // fullCalendar: {
```
Comment (Trolzie) [inline examples/group-bookings.htm]: what is this? are we saving it?

File: src/defaults.js [hunk 2]
```diff
@@ -138,6 +138,36 @@ var bookingConfirmDecline = {
...
+// Preset: bookingGraph = 'group_customer'
+var bookingGroupCustomer = {
+
+  timekitCreateBooking: {
+    graph: 'group_customer',
+    action: 'create',
+  },
+  localization: {
+    strings: {
+      successMessageBody: "You have reserved a seat at the booking. Check your email for further instructions. <br /><br />Have a great day!"
```
Comment (Trolzie) [inline src/defaults.js]: "You have reserved a seat at the booking." This sounds a little funny. why not keep it simple with something like:
"Your seat has been reserved, please check your email for instructions."

File: test/groupBookings.spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+        expect($('.bookingjs-form').hasClass('loading')).toBe(true);
+
+        setTimeout(function() {
+
+          expect($('.bookingjs-form').hasClass('success')).toBe(true);
+          expect($('.bookingjs-form-success-message')).toBeVisible();
+
+          var successMessage = $('.bookingjs-form-success-message').html();
+          var contains = successMessage.indexOf('You have reserved a seat at the booking') > -1;
```
Comment (Trolzie) [inline test/groupBookings.spec.js]: "You have reserved a seat at the booking". 
I'm pretty sure correct grammar would be: "You have reserved a seat for this booking"

File: src/defaults.js (hunk 2 above)
Comment (laander) [inline src/defaults.js]: Good catch. Re-worded it :+1:

File: test/groupBookings.spec.js (hunk 3 above)
Comment (laander) [inline test/groupBookings.spec.js]: Fixed

File: examples/group-bookings.htm (hunk 1 above)
Comment (laander) [inline examples/group-bookings.htm]: Ah, I forget to remove some local testing stuff. Cleaned it up now

File: src/defaults.js [hunk 4]
```diff
@@ -147,7 +147,7 @@ var bookingGroupCustomer = {
   },
   localization: {
     strings: {
-      successMessageBody: "You have reserved a seat at the booking. Check your email for further instructions. <br /><br />Have a great day!"
+      successMessageBody: "Your seat has been reserved and we've send you a confirmation by email. <br /><br />Have a great day!"
```
Comment (Trolzie) [inline src/defaults.js]: "we've send" -> "we've sent"

**Comment by Trolzie** [comment]
gj ♠️

**Pull request merged**